### PR TITLE
pdftohtml: add page

### PR DESCRIPTION
**Pull request by shellheim**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 23.02.0

### CODE REVIEW

**Comment by shellheim** [comment]
I didn't tick the [guidelines](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines) option because the program doesn't have long form options like `--ignore` and consequently, the page doesn't either.

File: pages/linux/pdftohtml.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# pdftohtml
+
+> Pdftohtml is a program to convert PDF files into HTML and XML and PNG images.
```
Comment (kant) [inline pages/linux/pdftohtml.md:3]: Pdftohtml is a program to convert PDF files into HTML, XML and PNG images.

**Review by kant — DISMISSED** [review]
Line 3: Pdftohtml is a program to convert PDF files into HTML, XML and PNG images.

**Comment by shellheim** [comment]
Well is does convert to it but so it can be viewed alongside the html/xml page not as a standalone image. But I will add that.

File: pages/linux/pdftohtml.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# pdftohtml
+
+> Pdftohtml is a program to convert PDF files into HTML, XML and PNG images.
+> More information: <https://linux.die.net/man/1/pdftohtml>.
```
Comment (kbdharun) [inline pages/linux/pdftohtml.md:4]:
```suggestion
> Program to convert PDF files into HTML, XML and PNG images.
> More information: <https://manned.org/pdftohtml.1>.
```
We use manned as a fallback when there isn't an official documentation page.

Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#more-information-links

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

> Btw @kant for general suggestions I would suggest going with the "Comment" option instead of using "request changes" especially with [new contributors](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/maintainers-guide.md). Welcome to maintaining tldr, feel free to drop by the chatroom if you have any queries regarding PRs.

File: pages/linux/pdftohtml.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`pdftohtml -i {{path/to/pdf}} {{path/to/html}}`
+
+- Generate a single HTML that includes all pdf pages:
+
+`pdftohtml -s {{path/to/pdf}} {{path/to/html}}`
+
+- Convert a pdf file to a xml file:
+
+`pdftohtml -xml {{path/to/pdf}} {{path/to/html}}`
```
[the remark this answers is not in the document]
Comment (shellheim) [reply pages/linux/pdftohtml.md:20]: that's way better, thanks!

File: pages/linux/pdftohtml.md (hunk 2 above)
Comment (shellheim) [reply pages/linux/pdftohtml.md:4]: Oh, I read the style guide but didn't see that, sorry.

**Review by kbdharun — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
Already approved 😄, here you go again thanks.

**Comment by shellheim** [comment]
you approved changes just a second before I clicked request review lol

File: pages/linux/pdftohtml.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# pdftohtml
+
+> Convert PDF files into HTML, XML and PNG images.
+> More information: <https://manned.org/pdftohtml.1>.
```
Comment (acuteenvy) [inline pages/linux/pdftohtml.md:4]:
```suggestion
> More information: <https://manned.org/pdftohtml>.
```

**Review by acuteenvy — APPROVED** [review]

File: pages/linux/pdftohtml.md (hunk 4 above)
Comment (shellheim) [reply pages/linux/pdftohtml.md:4]: what's the difference?

Comment (kbdharun) [reply pages/linux/pdftohtml.md:4]: > what's the difference? 

.1 isn't necessary in the URL if there isn't another command/repackage with the same name. The suggestion is a minor cosmetic fix.

Comment (shellheim) [reply pages/linux/pdftohtml.md:4]: Oh, got it.

**Pull request merged**